ci/build_ndcore.sh

```diff
@@ -12,7 +12,7 @@ fi
 
 # Get the common synapses
 COMMON_DIR=_common
-git clone git@bbpgitlab.epfl.ch:hpc/sim/models/common.git $COMMON_DIR  --depth=1
+[ -d _common ] || git clone git@bbpgitlab.epfl.ch:hpc/sim/models/common.git $COMMON_DIR  --depth=1
```

Comment:
I added this to make local dev faster. In CI it will always clone fresh from main

Reply:
I would prefer if `_common` got updated if it is already present.

As an unknowing user, I'd run the script and have a `_common` directory - now I let this sit for 6 months, do a `git pull`, and run the script again. If I don't notice `_common` and update it manually, it may be out of sync...

Finally, If you define `COMMON_DIR`, it should be used :)
```suggestion
[ -d "$COMMON_DIR" ] || git clone git@bbpgitlab.epfl.ch:hpc/sim/models/common.git "$COMMON_DIR"  --depth=1
```